web/server/go.md

```diff
@@ -0,0 +1,498 @@
+# Web Applications in Go
```

Comment:
I started writing this in the initial golang guide, but it seemed worth its own section. Feel free to suggest placement.